### PR TITLE
[usecase-webapi] Remove Fingerprint from the samples of usecase webapi

### PR DESCRIPTION
**Pull request by qiuzhong**
Fingerprint has been integrated to the demo-express, so remove it from
the samples of usecase-webapi-xwalk-tests as it'll be copied from the
demo-express repo each time.

BUG=https://crosswalk-project.org/jira/browse/CTS-615

### CODE REVIEW

**Comment by qiuzhong** [comment]
@Honry , remove fingerprint from usecase-webapi.

File: usecase/usecase-webapi-xwalk-tests/samples/Fingerprint/README.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# Fingerprint authentication feature example
-## Introduction
-This example is for testing the fingerprint authentication feature.
-## Prerequisites
-* Copy the index.html from [crosswalk-samples fingerpint sample](https://github.com/crosswalk-project/crosswalk-samples/fingerprint/www/index.html) to ```fingerprint``` directory.
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/samples/Fingerprint/README.md]: Please move this precondition to README.md file under usecase-webapi-xwalk-tests

Comment (qiuzhong) [inline usecase/usecase-webapi-xwalk-tests/samples/Fingerprint/README.md]: README.md for the test suite added.

**Comment by Honry** [comment]
LGTM

**Pull request merged**